drivers/frequency/adf4371/adf4371.c

```diff
@@ -696,8 +720,17 @@ int32_t adf4371_clk_recalc_rate(struct adf4371_dev *dev, uint32_t chan,
  * @param rounded_rate - The closest possible rate of desired rate.
  * @return 0 in case of success, negative error code otherwise.
  */
-int32_t adf4371_clk_round_rate(struct adf4371_dev *dev, uint64_t rate,
-			       uint64_t *rounded_rate)
+int32_t adf4371_clk_round_rate_chan(struct adf4371_dev *dev, uint64_t rate,
```

Comment:
hmm, now channel input parameter? (as suggested by the function name). also the function header comment i think needs some edits.

Reply:
Since the function was originally named with suffix _chan, I expected a channel parameter to be added, which would also require updating the header comment. However, that’s no longer the case.